### PR TITLE
Enable `localStorage` polyfill in Opera Mini

### PR DESCRIPTION
**Pull request by mathiasbynens**
#498.

### CODE REVIEW

**Comment by triblondon** [comment]
Thanks @mathiasbynens.  Can I clarify what versions mean in relation to mini?  Is everyone on the latest version regardless of the age of their client, or is the client version used to determine which version of the server component to run?

So in practical terms, what does it mean to say 'users of Opera mini 5'?

**Comment by mathiasbynens** [comment]
cc @brucelawson

**Comment by brucelawson** [comment]
finding out definitively from the Mini P.M., will report back.

**Comment by brucelawson** [comment]
Apologies for the delay, but I wanted to be absolutely sure, so asked Opera Mini PM and engineers.

All Opera Mini clients use the same server version and that
also means they all use the same Presto version. JavaScript and CSS
support is the same for all clients, with the exception that user may
enable Small Screen Rendering which removes a lot of CSS rules. They
can also use different viewport settings, but the same holds for
normal browsers. Version numbers effectively refer to different
versions of the client UI and capabilities (whether it supports
'night mode', private tabs, progressive download) etc.

**Comment by brucelawson** [comment]
Added a note to https://dev.opera.com/articles/making-sites-work-opera-mini/ prompted by this Q. Addition reads:

Because Opera Mini renders on our servers, all Opera Mini clients use the same server version, so all use the same Presto version. JavaScript and CSS support is therefore identical on all clients, with the exception that user may enable Small Screen Rendering which removes a lot of CSS rules.

Version numbers refer to different versions of the client UI and capabilities (whether it supports night mode, private tabs, progressive loading, etc.)

Opera Mini 4.x clients load the whole transcoded page at once, as they have no progressive loading. This slightly reduces the time allowed for JavaScript execution (including XHRs). With newer clients, the transcoder is more generous, since it is able to send the compressed page to the client in chunks.

**Comment by triblondon** [comment]
OK, I think I've gotten my head around this:
- When I request a polyfill bundle from the service through Opera Mini on a generic fruit based telecommunications device (model 6), I get identified as `op_mini/12.0.0`
- According to the [Opera Mini request headers doc](https://dev.opera.com/articles/opera-mini-request-headers/), this appears to be parsing correctly
- The last server upgrade was in March 2015, and all Opera Mini users will be on the same version
- This PR targets Opera Mini up to v11, which no-one will be using

So it seems to make sense to me for all our Opera Mini targets to be `*`, unless the polyfill is for something related to the core rendering engine, in which case we can set it to the presto version in which that feature is introduced and then we should automatically stop serving the polyfill once the Big Machine is updated to a version of Presto greater than that.

But for localStorage, it's presumably not in the rendering engine but in your JS engine, which I imagine is very custom for Mini and not comparable with a desktop version series.

So, TLDR: this should probably be `*`, not `5 - 11`.  WDYT?

**Comment by triblondon** [comment]
Paging @brucelawson and @mathiasbynens see update above (soz for not mentioning you in it)

**Comment by mathiasbynens** [comment]
+1 to using `*` instead of `5 - 11`.

**Comment by triblondon** [comment]
@mathiasbynens can you change it then?  This PR in in your fork.

**Comment by mathiasbynens** [comment]
Done.
